src/Symfony/Bundle/FrameworkBundle/Resources/config/debug_prod.xml

```diff
@@ -14,13 +14,15 @@
         <service id="debug.debug_handlers_listener" class="Symfony\Component\HttpKernel\EventListener\DebugHandlersListener">
             <tag name="kernel.event_subscriber" />
             <tag name="monolog.logger" channel="php" />
+            <tag name="monolog.logger" channel="deprecation" />
             <argument>null</argument><!-- Exception handler -->
-            <argument type="service" id="logger" on-invalid="null" />
+            <argument type="service" id="monolog.logger.php" on-invalid="null" />
```

Comment:
It will break apps, that is use monolog as package for logging.
Isn't it?

To avoid errors they should declare "monolog" services with alias to main logger?

Reply:
No (at least with the current proposed implementation) it should not break the app. If there's no `monolog.logger.deprecation` service, deprecations will be logged using `monolog.logger.php`.